docs/dev-guides/tutorials/adding-ownerships.md

```diff
@@ -0,0 +1,143 @@
+# Adding Ownerships
+
+## Why Would You Add Ownerships? 
+Assigning an owner to an entity helps to establish accountability for the metadata and collaborating as a team. 
+If there are any issues or questions about the data, the designated owner can serve as a reliable point of contact.
+
+## Pre-requisites
+For this tutorial, you need to deploy Datahub Quickstart and ingest sample data. 
+For detailed information, please refer to [Prepare Local Datahub Environment](http://yoonhyejin.github.io/datahub-forked/docs/dev-guides/tutorials/references/prepare-datahub).
+
+:::note
+Adding ownerships assumes that you already have a dataset and ownerships on your datahub.
+If you try to manipulate with entities that does not exist, it might return errors or failed to reference them.
+:::
+
+
+## Add Ownerships With Python SDK
+
+Following codes add an owner named `bfoo` to a hive dataset named `fct_users_created`.
+You can refer to a full code in [dataset_add_column_ownership.py](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/examples/library/dataset_add_owner.py).
+```python
+# inlined from metadata-ingestion/examples/library/dataset_add_column_ownership.py
```

Comment:
i'm going to work on an "inlining" plugin that should reduce the copy-paste here. I definitely worry about these examples going stale as the sdk evolves.